### ISSUE TITLE
how to predict longer sequences?

### ISSUE THREAD

**Issue by bigfeetsmalltone** [seominseok0429/Implicit-Stacked-Autoregressive-Model-for-Video-Prediction#5]
Good work, but I have some questions. In the code, the input length equals the output length, namely 10 -> 10. How to predict longer sequences, such as 10 -> 30? Besides, the proposed model employs the predicted frames rather than ground truth to produce the next frame in the training phase, which causes inconsistency between the code and the paper. I have noticed that, in the previous version of the code, the model employs ground truth randomly masked to make predictions in the training phase.

**Comment by seominseok0429** [comment]
Thank you for your interest!

1. Our method can predict longer sequences by increasing the number of future queues or performing autoregressive inference. However, as the limitations in our paper, the training time increases linearly as the number of sequences increases. **I recommend that you use our method to be autoregressive.**

2. That's right. In our first version code, only the training code was uploaded. The code uploaded on April 5th was the code that can train and test. However, the code uploaded now matches IAM4VP in all its structure, but the training strategy has changed. (for fast convergence)
